coralproject/talk
# Support multisite

## What does this PR do?
- **updates install process (CORL-816)** installation process will now create a single site and community
- **add a new site (CORL-812)** Add a new site form creates new site and sets multisite: true on tenant
- **view sites (CORL-813)** lists sites with links to edit
- **update sites (CORL-814)** update basic site settings
- **filter stories by site (CORL-841)** adds filter dropdown to stories table
- **filter moderation by site (CORL-817)** adds filter dropdown to moderation search bar
- migration to make existing tenants compatible
- stream-side updates to determine site of a story from domain
- adds new global notification component

## How do I test this PR?
- test migration by trying to run with existing tenant 
- try to add a new site, make sure moderation, stories table, and stream still work
- test installation process against empty db
